runbotics/runbotics-orchestrator-ui/src/src-app/views/process/AddProcess.tsx

```diff
@@ -91,8 +93,15 @@ const AddProcessDialog: FC<AddProcessDialogProps> = ({
         }
     };
 
+    const handleKeypress = (e: React.KeyboardEvent) => {
```

Comment:
Here instead of using keypress === enter and then submit, just use form (html one) element and add on submit to it